extension/docs/HowToRun.md

```diff
@@ -148,6 +149,10 @@ Other configurations can be set as direct child attributes in `ADYEN_INTEGRATION
 | `basicAuth`                  | Boolean attribute to enable/disable basic authentication to prevent unauthorized 3rd-party from accessing extension endpoint                                                       | NO       | false                                                                                                                        |
 | `removeSensitiveData`        | Boolean attribute. When set to "false", Adyen fields with additional information about the payment will be saved in the interface interaction and in the custom fields.            | NO       | true                                                                                                                         |
 
+### External file configuration
+
+In case you have a huge configuration that reaches above the environment limits (e.g the total size of all environment variables on AWS Lambda can't exceed 4 KB.), you could use the external file configuration, with that the extension module will look the `.extensionrc` file in the extension folder, the file should contain the same content as it's defined with environment variable.
```

Comment:
```suggestion
In case you have a huge configuration that reaches above the environment limits (e.g the total size of all environment variables on AWS Lambda can't exceed 4 KB.), you could use the external file configuration. The extension module will look for the `.extensionrc` file in the `extension` folder. The file should contain the same content as it's defined with environment variable.
```

Reply:
So you should put the JSON in the file, right? Not stringified json or sth like that? Can you add this to the doc?

Reply:
Can you also add a note that env var will take precedence over the external config file?

Reply:
https://github.com/commercetools/commercetools-adyen-integration/pull/870/commits/223c2669c852b95296696e4c24e4562f71b4f132